src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts

```diff
@@ -150,17 +150,9 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
       const upcsRegex = new RegExp('^(?=.*[A-Z])');
       const charRegex = new RegExp('^(?=.{8,})');
       const numRegex = new RegExp('^(?=.*[0-9])');
-      const specRegex = new RegExp('^[^<>{}\'\"/|;:.,~!?@#$%^=&*\\]\\\\()\\[¿§«»ω⊙¤°℃℉€¥£¢¡®©_+]*$');
-      if (!charRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.charError;
-      } else if (!lwcsRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.lwcsError;
-      } else if (!upcsRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.upcsError;
-      } else if (!numRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.numError;
-      } else if (specRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.specError;
+      const specRegex = new RegExp('^[^<>{}\'\"/|;:.\ ,~!?@#$%^=&*\\]\\\\()\\[¿§«»ω⊙¤°℃℉€¥£¢¡®©_+]*$');
+      if (!charRegex.test(val) || !lwcsRegex.test(val) || !upcsRegex.test(val) || !numRegex.test(val) || specRegex.test(val)) {
+        this.passwordError = this.resourceService.frmelmnts.lbl.passwd;
```

Comment:
please use the _.get to get the Json opject

Reply:
Done